modules/playing-cards/assets/scss/frontend.scss

```
@@ -0,0 +1,93 @@
+$suit_colors: (
+    b: #1e1e1e,
+    r: #F44336,
+);
+
+$borderRadius: 8px;
+
+ul.elementor-card-list-items {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: var(--e-playing-cards-gap);
```

If for some reason the editor didn't generate CSS it's going to fail, it's better to put a default value.
```suggestion
    gap: var(--e-playing-cards-gap, default value);
```